app/jobs/disability_claim/document_upload.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class DisabilityClaim::DocumentUpload < ActiveJob::Base
+  def perform(filename, auth_headers, user_uuid, claim_id, tracked_item_id)
+    client = EVSS::DocumentsService.new(auth_headers)
+    uploader = DisabilityClaimDocumentUploader.new(user_uuid, tracked_item_id)
+    uploader.retrieve_from_store!(filename)
+    client.upload(filename, uploader.read, claim_id, tracked_item_id)
```

Comment:
There is specs for both (the worker itself, and that the request to the endpoint enqueues the correct job). I can add a check for the arguments as there are enough here that ensuring order would probably be beneficial (or switching to keyword arguments)